dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -2709,9 +2705,7 @@ const CalpontSystemCatalog::RID CalpontSystemCatalog::constraintColRID(const Tab
     aTableColName.schema = tableColName.schema;
     aTableColName.table = tableColName.table;
     aTableColName.column = tableColName.column;
-    transform( aTableColName.schema.begin(), aTableColName.schema.end(), aTableColName.schema.begin(), to_lower() );
-    transform( aTableColName.table.begin(), aTableColName.table.end(), aTableColName.table.begin(), to_lower() );
-    transform( aTableColName.column.begin(), aTableColName.column.end(), aTableColName.column.begin(), to_lower() );
+    boost::algorithm::to_lower(aTableColName.column);
```

Comment:
I know this code above is commented out, so the deletions above will have no effect. But in case this is activated in future, we will lose the changes where we might need to apply the to_lower conversions based on the value of `lower_case_table_names`

Reply:
Absolutely true. There are tons of commented out code having to do with indexes and constraints. I started down the path of changing them (removing the transforms), but gave up because of time and inability to test commented code, so I never added the if (lower_case_table_names) stuff in. If we ever enable these blocks, they will most likely be re-written anyway, so we just need to be aware.

Reply:
Maybe, but to retrofit would take a bit. Time constraints...